ztp/source-crs/TunedPerformancePatch.yaml

```diff
@@ -25,6 +25,7 @@ spec:
         [service]
         service.stalld=start,enable
         service.chronyd=stop,disable
+        service.kubens=start,enable
```

Comment:
In the prior implementation the service gets dropped in by MC during installation and would be present from then on. With this implementation the service doesn't get enabled until this tuned patch is applied at day-2 (much later in the process). Perhaps that is OK since this will still be before a workload is applied (and the mount namespace hiding is helping reduce CPU with large numbers of pods) but there is a period of time where the system will be running without it. 
Is there a way to enable it at installation time?

Reply:
Yep it's possible with a MC

Reply:
I think we had originally talked about enabling it here because it might have been a better way to have the 2 different implementations coexist - But since we're ripping out the old one entirely, maybe going with the MC-based enable would be better than the NTO for the reason @imiller0 points out.